# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lvm/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


The usual gcc-3.4.0 fixes.

  - Rene Rebe <rene@exactcode.de>

diff -ur 1.0.8-orig/tools/lib/pv_read_all_pv_of_vg.c 1.0.8/tools/lib/pv_read_all_pv_of_vg.c
--- 1.0.8-orig/tools/lib/pv_read_all_pv_of_vg.c	2003-09-03 17:16:08.000000000 +0200
+++ 1.0.8/tools/lib/pv_read_all_pv_of_vg.c	2004-05-02 23:31:36.000000000 +0200
@@ -130,6 +130,7 @@
             }
             pv_this[p] = NULL;
 uuid_check_end:
+	    ((void)0);
          }
          for ( pp = 0; pp < p - 2; pp++) {
             if ( pv_this[pp] == NULL) {
diff -ur 1.0.8-orig/tools/lib/pv_read_uuidlist.c 1.0.8/tools/lib/pv_read_uuidlist.c
--- 1.0.8-orig/tools/lib/pv_read_uuidlist.c	2003-02-06 16:05:34.000000000 +0100
+++ 1.0.8/tools/lib/pv_read_uuidlist.c	2004-05-02 23:32:29.000000000 +0200
@@ -50,7 +50,7 @@
    static char *this_pv_uuidlist = NULL;
    int num;
 
-   debug_enter(__FUNCTION__ " -- CALLED with %s\n", pv ? pv->pv_name : "NULL");
+   debug_enter("%s -- CALLED with %s\n", __FUNCTION__, pv ? pv->pv_name : "NULL");
 
    if (pv == NULL || pv_uuidlist == NULL)
       return -LVM_EPARAM;
diff -ur 1.0.8-orig/tools/lib/pv_write_uuidlist.c 1.0.8/tools/lib/pv_write_uuidlist.c
--- 1.0.8-orig/tools/lib/pv_write_uuidlist.c	2003-02-06 16:05:34.000000000 +0100
+++ 1.0.8/tools/lib/pv_write_uuidlist.c	2004-05-02 23:34:26.000000000 +0200
@@ -71,7 +71,8 @@
 
             /* Create/fix UUIDs for any PVs that need it */
             if (lvm_check_uuid(pv->pv_uuid) < 0) {
-               debug(__FUNCTION__ " -- creating new UUID for PV %s\n",
+               debug("%s -- creating new UUID for PV %s\n",
+		     __FUNCTION__,
                      pv->pv_name);
                memset(pv->pv_uuid, 0, sizeof(pv->pv_uuid));
                memcpy(pv->pv_uuid, lvm_create_uuid(UUID_LEN), UUID_LEN);
